Sources/OktaIdxAuth/Internal/Requests/SocialAuth.swift

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2021, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import AuthenticationServices
+import Foundation
+import OktaIdx
+
+extension OktaIdxAuth.Implementation.Request {
+    @available(iOSApplicationExtension 13.0, *)
+    class SocialAuthenticate: Request<Response>, OktaIdxAuthRemediationRequest {
+        private var webAuthSession: ASWebAuthenticationSession?
```

Comment:
I would like to use one object and not separate iOS12 class. But Swift doesn't support stored property with @available attribute. The error is thrown.